blocks/v2-columns/v2-columns.css

```diff
@@ -0,0 +1,66 @@
+.v2-columns {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.v2-columns .v2-columns-image,
+.v2-columns .v2-columns-content {
+  max-width: 343px;
+}
+
+.v2-columns .v2-columns-content {
+  padding: 40px 0;
+}
+
+.v2-columns .v2-columns-content p.v2-columns-content__eyebrow {
+  font-family: var(--ff-accents-regular);
+  font-size: var(--accent-2-font-size);
+  line-height: var(--accent-2-line-height);
+  margin: 0;
+}
+
+.v2-columns .v2-columns-content h1.v2-columns-content__headline {
```

Comment:
Avoid using tags `h1`, `p` in CSS as much as possible. 
What will happen if the editor decides to use `h2` instead of `h1`? The styles won't be applied. But if you just use the class, any tag containing the class will be styled.